### ISSUE TITLE
Consider auto-spec of AFK players at !start

### ISSUE THREAD

**Issue by KingRaptor** [ZeroK-RTS/Zero-K-Infrastructure#1832]
This request shows up from time to time, e.g. [here](http://zero-k.info/Forum/Post/174861#174861) and today in #zk channel. 

For optimal results, AFK should detect out-of-lobby status, e.g. someone who is using web browser at that moment should not be considered AFK. Also fix !ring and make Springie use it on AFKers when a !start vote occurs.

There's a chance that AFK people will return when the game starts, but if more than one of them fail to do so (especially if they're on the same team) I expect people will get annoyed in pretty short order. I think it's not unreasonable to expect people who want to play to be actually present.

**Comment by sprunk** [comment]
This had been the case for quite a long time so it's probably a good idea to dig up the change and figure out the reasons for removal.

**Comment by DeinFreund** [comment]
Currently `!poll start` doesn't require the votes of afkers. This was done assuming they are not included in the game. Worst case example: 1v1 with an afk player can be started solely by the non-afk player doing `!poll start`.

**Comment by DeinFreund** [comment]
It seems autospec never existed since operation murder.

**Comment by Licho1** [comment]
As KR said first AFK needs to detect out of lobby state. And even that it is HIGHLY questionable. Why to automate it? People can spec

**Comment by DeinFreund** [comment]
But start vote now has ring, if somebody is interested in playing he'll tab into the game before the start vote completes. We can also just leave speccing to the players and close this.

**Comment by Licho1** [comment]
Nah, i often join cluster games and just code until game appears. Dont care about votes, it will start even without my clicking.

**Comment by sprunk** [comment]
Thread again. https://zero-k.info/Forum/Thread/25549

**Comment by sprunk** [comment]
Thread again. https://zero-k.info/Forum/Thread/25817

**Comment by Histidine91** [comment]
Compromise solution that doesn't require changing AFK detection: !start is delayed by 10-15 seconds. At the start of the timer, it rings everyone with a loud Chobby notification, when it ends run !specafk and start the game.

**Comment by sprunk** [comment]
Sounds a bit like what the MM does with the accept popup.

**Comment by cleanrock** [comment]
As I think Licho said a few times, AFK should be detected system wide, i.e. no keyboard or mouse input for X minutes.
This is how flobby works (5 minutes) and the few flobby users have not complained about it.
